### DISCUSSION TITLE
[2021.09.13] 이진탐색 알고리즘 시작하기

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#124] [Algorithm]
# 이진 탐색 알고리즘을 구현하라

#### ❗[동영상 참고] (https://www.youtube.com/watch?v=94RC-DsGMLo)

**Comment by pyunji** [comment]
[이진탐색 알고리즘 시작하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/Algorithm/Basic_BinarySearch.java)

**Comment by hyk339** [comment]
[이진탐색 알고리즘 시작하기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/dataStructure/Basic_BinarySearch.java)

**Comment by leechang4** [comment]
[이진탐색 알고리즘 시작하기 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/DataStructure/Basic_BinarySearch.java)

**Comment by chaeyeon4u** [comment]
[이진탐색 알고리즘 시작하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/dataStructure/BinarySearchBasic.java)
